Add theme ananthb/jughead

ananthb opened the pull request:
ananthb/jughead is a lightweight blog theme for Hugo.

After you have read the [instructions for adding a theme](https://github.com/gohugoio/hugoThemesSiteBuilder/blob/main/README.md#adding-a-theme), please make sure:

- [X] your theme.toml [is complete](https://github.com/gohugoio/hugoThemesSiteBuilder/blob/main/README.md#theme-configuration)
    - [X] name
    - [X] license
    - [X] licenselink
    - [X] description
    - [X] homepage
    - [X] demosite (if one exists)
    - [X] tags
    - [X] features
    - [X] authors/author (depending on whether the theme has multiple or a single author)
    - [X] original (if the theme is a fork)
- [X] you're using [absolute paths for images](https://github.com/gohugoio/hugoThemesSiteBuilder/blob/main/README.md#use-absolute-paths-for-images) in your README
- [X] you've got [appropriate thumbnail and screenshot images](https://github.com/gohugoio/hugoThemesSiteBuilder/blob/main/README.md#media)
- [X] your theme comes with an [appropriate license](https://github.com/gohugoio/hugoThemesSiteBuilder/blob/main/README.md#2-license)

hugo-sid commented:
Thanks for adding your theme.

From the README:

> 1. Forks must be notably different
A theme based on an existing Hugo theme (aka a fork) must be notably different for it to be considered a separate theme altogether. In such cases, you should list few arguments in README.md file mentioning why your theme should be included.

> The definition of notably different can be subjective, but in most cases, it should be clear. Changing a few colors or making a few style changes, for example, does not result in a notably different theme. It would be better if you submit a [pull request](https://docs.github.com/en/pull-requests) to the original theme to include your proposed changes.

Can you list a few arguments in README.md which shows how your theme is different than the fork?

ananthb commented:
Sure thing. 

On the surface it's similar, but visually I've fixed issues with incorrect spacing, colour contrast issues, and colour palette accessibility issues.

A lot has changed internally in how the theme loads different features. I've re-worked how comments work and I've added support for diagrams, charts, etc.

I've also added better support for Hugo asset pipelines which makes sites load faster and work better. I've also started preliminary work for internationalisation support.

I plan on going a different direction visually now that the internals have largely been figured out.

ananthb commented:
I'll add this note to the readme shortly.

ananthb commented:
@hugo-sid I've added a note to my theme's README talking about the reason behind the fork, what's different, and about future plans as well.

ananthb commented:
@hugo-sid my theme was (unintentionally I hope) removed in this commit: https://github.com/gohugoio/hugoThemesSiteBuilder/pull/473. How do I restore it?
